.travis.yml

```diff
@@ -40,147 +40,120 @@ language: python
   - &hetzner-cloud-cleanup
     after_script:
       - ./contrib/tokenmgmr.py remove --driver hetznercloud
-  - &env-functional
```

Comment:
Don't delete this, specify the new list style syntax here.

There is a reason this is here. Changing it later is repeating ourselves everywhere.

Reply:
There is a reason why I removed it, we cannot use it any longer. I will explain:

Because `env:` is no longer a dictionary and now is a list, we no longer can use YAML anchors to combine dictionaries. Lists cannot be combined like dictionaries. That is documented limitation of `<<:` operator.

I am for deduplicating code and avoiding repetition. Still, in this case the `deduplicated` code had the same length as the non-deduplicated one (count lines before/after). Mainly we added a new layer which introduced bugs.

If you find another way to reorganize the file in which `env:` blocks are lists, I would be more than happy to take it.

Reply:
What is critical in this change is to assure that the triggered jobs use the right argumens as we do not want to introduce typos.

Reply:
You'd save time if you put this in the description.

The PR just looks like a refactoring without this explanation.